gnd/src/main/res/layout/record_details_frag.xml

```diff
@@ -14,79 +14,88 @@
-      app:cardElevation="@dimen/toolbar_elevation">
-
-      <LinearLayout
+      android:layout_height="match_parent"
+      android:orientation="vertical">
+  
+      <!-- CardView is used to create the drop shadow effect -->
+      <android.support.v7.widget.CardView
+        android:id="@+id/record_details_header"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical">
-
-        <com.google.android.gnd.ui.common.TwoLineToolbar
-          android:id="@+id/record_details_toolbar"
-          android:layout_width="match_parent"
-          android:layout_height="?attr/actionBarSize"
-          android:layout_gravity="top"
-          android:theme="@style/WhiteToolbarTheme"/>
-
-        <TextView
-          android:id="@+id/form_name"
-          android:textStyle="bold"
+        app:cardCornerRadius="0dp"
+        app:cardElevation="@dimen/toolbar_elevation">
+  
+        <LinearLayout
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:paddingBottom="6dp"
-          android:background="@color/colorBackground"
-          android:textAlignment="center"
-          android:textAllCaps="true"
-          android:textColor="@color/colorForeground"
-          android:textSize="12sp"
-          />
-
-      </LinearLayout>
-
-    </android.support.v7.widget.CardView>
-
-    <ProgressBar
-      android:id="@+id/record_details_progress_bar"
-      style="?android:attr/progressBarStyle"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_gravity="center"
-      android:padding="48dp"/>
-
-    <ScrollView
-      android:layout_width="match_parent"
-      android:layout_height="match_parent"
-      android:layout_below="@id/record_details_header"
-      android:background="@color/colorBackground">
-
-      <LinearLayout
-        android:id="@+id/record_details_layout"
-        android:layout_width="match_parent"
+          android:orientation="vertical">
+  
+          <com.google.android.gnd.ui.common.TwoLineToolbar
+            android:id="@+id/record_details_toolbar"
+            android:layout_width="match_parent"
+            android:layout_height="?attr/actionBarSize"
+            android:layout_gravity="top"
+            android:theme="@style/WhiteToolbarTheme"/>
+  
+          <TextView
+            android:id="@+id/form_name"
+            android:textStyle="bold"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingBottom="6dp"
+            android:background="@color/colorBackground"
+            android:textAlignment="center"
+            android:textAllCaps="true"
+            android:textColor="@color/colorForeground"
+            android:textSize="12sp"
+            />
+  
+        </LinearLayout>
+  
+      </android.support.v7.widget.CardView>
+  
+      <ProgressBar
```

Comment:
Does it make more sense for this to live at the top level of the layout? Does it still work there?

Reply:
Think that should be fine!